docs/user-guide/dream/dream-powder-reduction.ipynb

```diff
@@ -94,7 +94,10 @@
     "workflow[DspacingBins] = sc.linspace(\"dspacing\", 0.3, 2.3434, 201, unit=\"angstrom\")\n",
     "\n",
     "# Do not mask any pixels / voxels:\n",
-    "workflow = powder.with_pixel_mask_filenames(workflow, [])"
+    "workflow = powder.with_pixel_mask_filenames(workflow, [])\n",
+    "\n",
+    "# Enable subtraction by an empty can measurement:\n",
+    "powder.correction.add_empty_can_subtraction(workflow)"
```

Comment:
I don't like this mechanism. I think a dedicated class for powder workflows would help here.